### PR TITLE
updating jenkins plugin to include link to jenkins job and send more …

### PR DESCRIPTION
**Pull request by rdhanoa**
…info to jenkins

@zendesk/runway

### CODE REVIEW

File: plugins/jenkins/app/helpers/jenkins_helper.rb [hunk 1]
```diff
@@ -17,15 +17,20 @@ def jenkins_status_panel(deploy, jenkins_job)
...
+      jenkins_job.update_attributes!(attributes)
     end
 
     status = mapping.fetch(jenkins_job_status)
     status_message = result.fetch(jenkins_job_status)
 
     content = "Jenkins build #{jenkins_job.name} for #{deploy.stage.name} #{status_message}."
-    content_tag :div, content.html_safe, class: "alert alert-#{status}"
+    div = content_tag :div, content.html_safe, class: "alert alert-#{status}"
+    content_tag :a, div, href: jenkins_job_url, target: "_blank"
```
Comment (grosser) [inline plugins/jenkins/app/helpers/jenkins_helper.rb]: you can use blocks to make the structure easier to see:
```Ruby
content_tag :a, href: jenkins_job_url, target: "_blank" do
  content_tag :div, content.html_safe, class: "alert alert-#{status}"
end
```

**Comment by rdhanoa** [comment]
will add/update tests tomorrow

File: plugins/jenkins/app/models/samson/jenkins.rb [hunk 2]
```diff
@@ -15,7 +15,7 @@ def initialize(job_name, deploy)
 
     def build
       opts = {'build_start_timeout' => 60}
-      client.job.build(job_name, {'buildStartedBy' => deploy.user.name, 'originatedFrom' => deploy.stage.name}, opts).to_i
+      client.job.build(job_name, {'buildStartedBy' => deploy.user.name, 'originatedFrom' => deploy.project.name + '_' + deploy.stage.name + '_' + deploy.reference}, opts).to_i
```
Comment (grosser) [inline plugins/jenkins/app/models/samson/jenkins.rb]: maybe a new line with `originated_from = all this logic`

**Comment by grosser** [comment]
:+1:

**Comment by grosser** [comment]
maybe a new test ?

**Pull request merged**